selene_sdk/samplers/online_sampler.py

```diff
@@ -188,16 +175,35 @@ def __init__(self,
                     self.modes, mode))
         self.mode = mode
 
-        self.surrounding_sequence_radius = int(
-            surrounding_sequence_length / 2)
+
         self.sequence_length = sequence_length
-        self.bin_radius = int(center_bin_to_predict / 2)
-        self._start_radius = self.bin_radius
-        if center_bin_to_predict % 2 == 0:
-            self._end_radius = self.bin_radius
+        window_radius = int(self.sequence_length / 2)
```

Comment:
like this - why should window radius be self.sequence_length / 2 ? and that's different from start and end bin radius? I'm very confused about all of these hahah that's why i'm assuming it's from the multibin code?

Reply:
bin_radius is the radius for the region to derive targets from and window_radius is the radius of the sequence. I made these change so that the region to derive targets can be support using an interval specified by `center_bin_to_predict` (and the interval does not have to be at the center)